src/events/issues.js

```diff
@@ -8,8 +8,9 @@ exports.run = (client, payload) => {
   const repoOwner = repository.owner.login;
   const repoName = repository.name;
   const l = payload.label;
+  const check = client.cfg.activity.issues.pullRequests;
 
-  if (client.cfg.activity.issues.inProgress) {
+  if (client.cfg.activity.issues.inProgress || check.needsReview.label) {
```

Comment:
No changes should be in this file.

Reply:
Yes, definitely. This change is the remnant from an earlier commit which I forgot to revert back while pushing new commits. Changing it now, thanks!